src/plugins/index_pattern_management/public/components/edit_index_pattern/edit_index_pattern.tsx

```diff
@@ -121,7 +121,9 @@ export const EditIndexPattern = withRouter(
     const refreshFields = () => {
       overlays.openConfirm(confirmMessage, confirmModalOptionsRefresh).then(async (isConfirmed) => {
         if (isConfirmed) {
-          await indexPattern.refreshFields();
+          // todo catch error as in index_pattern
```

Comment:
Is this TODO a leftover task from this PR, or for future reference?

Reply:
ah, thought I got that one - addressed.